### ISSUE TITLE
I missing the option to remove installed plugins in JupyterLab3. 

### ISSUE THREAD

**Issue by soerendip** [jupyterlab/jupyterlab#10154]
I only see an about button in JupyterLab 3.0.9.

![image](https://user-images.githubusercontent.com/3391614/116003954-b3455700-a5bd-11eb-9607-b8c2aec346a2.png)

How can remove them now?

**Comment by agoose77** [comment]
### Some History
The old JupyterLab 2.x supported extensions by requiring the user to have `npm` installed in order to update the JupyterLab application build every time an extension was included or removed. This is what the extension manager was used for - searching for JupyterLab extensions on `npmjs.com`, and [un]installing these extensions.

JupyterLab 3.x uses [module federation](https://webpack.js.org/concepts/module-federation/) to allow JupyterLab to load pre-built extensions without re-building. It is still possible to install extensions that require this re-build step, but increasingly developers are making use of the "pre-built" feature.

This now means that the act of installing / uinstalling an extension is not performed by JupyterLab. JupyterLab just loads the modules that it finds in a special location. To install an extension, package managers like `pip` just need to put these modules in the right place. Therefore, JupyterLab is no longer responsible (or even able) for safely uninstalling certain extensions. That is why when you click the `About` button in the extension manager, you see a message like this

![image](https://user-images.githubusercontent.com/1248413/116042782-bba59c80-a666-11eb-8448-c8c8a4625821.png)

which states 
> Use your Python package manager (pip, conda, etc.) to uninstall the package bqplot

### TL;DR
JupyterLab doesn't manage pre-built extensions, so you should uninstall these using the same tools you installed them with e.g. conda, pip,.
